app/views/home/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div>
+  <h1>Inimitable</h1>
+  <div>
+    <h2>Your Poems</h2>
+    <% if @poems.present? %>
+      <dl>
+        <% @poems.each do |poem| %>
+          <dt><%= poem.title %></dt>
+          <dd><%= poem.status %></dd>
+          <dd><%= poem.updated_at %></dd>
+        <% end %>
+      </dl>
+    <% else %>
+      <p>Looks like you don't have any poems yet.</p>
+    <% end %>
+    <%= button_to t("poems.actions.new"), new_poem_path, method: :get %>
```

Comment:
Any reason you used a `button_to` here? Is it for the visual look? Functionally, a link would be a better semantic HTML element to use. Consider using a `link_to` but using CSS to style it to look like a button.